test/verify/check-machines

```diff
@@ -138,7 +138,12 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
             b.wait_not_present(vm_row)
 
     def togglePoolRow(self, poolName, connectionName="system"):
```

Comment:
Not super sure about this last commit. It may be even flakier if we don't wait for the correct status in the first place. I think it would be better if we would give `toggle*Row` another argument like `close=False` and based on that we would first wait for the correct status, then click and then wait for the correct new status.